.gitignore

```diff
@@ -19,14 +19,17 @@ cms/envs/private.py
 .redcar/
 codekit-config.json
 .pycharm_helpers/
-/_mac/* 
```

Comment:
Reviewers: These lines lost a space at the end of the line, although I'm not sure why it isn't showing this in Github the same way line 29/32 does below.